rnd/autogpt_server/autogpt_server/blocks/basic.py

```diff
@@ -238,15 +194,93 @@ def run(self, input_data: Input) -> BlockOutput:
         yield "result", input_data.value
 
 
-class OutputBlock(InputOutputBlockBase[Any]):
+class OutputBlock(Block):
+    """
+    Records the output of the graph for users to see.
+
+    Attributes:
+        recorded_value: The value to be recorded as output.
+        name: The name of the output.
+        description: The description of the output.
+        fmt_string: The format string to be used to format the recorded_value.
+
+    Outputs:
+        output: The formatted recorded_value if fmt_string is provided and the recorded_value
+                can be formatted, otherwise the raw recorded_value.
+
+    Behavior:
+        If fmt_string is provided and the recorded_value is of a type that can be formatted,
+        the block attempts to format the recorded_value using the fmt_string.
+        If formatting fails or no fmt_string is provided, the raw recorded_value is output.
+    """
+
+    class Input(BlockSchema):
+        recorded_value: Any = SchemaField(
+            description="The value to be recorded as output."
+        )
+        name: str = SchemaField(description="The name of the output.")
+        description: str = SchemaField(description="The description of the output.")
+        fmt_string: str = SchemaField(
```

Comment:
You have to set a default value for fmt_string to None or set fmt_string to advanced=True